### PR TITLE
Updating PCIe gathering to get_pci_address

### PR DESCRIPTION
**Pull request by dkosteck**
Addressing https://github.com/redhat-partner-solutions/rhel-sriov-test/issues/149 and https://github.com/redhat-partner-solutions/rhel-sriov-test/issues/91
Updating documentation
Removing pci fields from testbed.yaml
Refactoring test cases to use get_pci_address
Refactoring container commands to utils
Update tests appropriately

### CODE REVIEW

File: sriov/tests/common/test_utils.py [hunk 1]
```diff
@@ -26,17 +26,6 @@
 )
 
 
-def test_get_pci_address(dut, settings):
```
Comment (dkosteck) [inline sriov/tests/common/test_utils.py:29]: Note that there is no real way I could think of to test this now. There is no field in the yaml for PCI, so there is nothing to check against. Perhaps another Linux command? That's still making assumptions, so perhaps we don't have this test.

**Comment by dkosteck** [comment]
@jianzzha Maybe we can reconcile this PR and your PR: https://github.com/redhat-partner-solutions/rhel-sriov-test/pull/145
For instance you can take a look at how I moved the container declarations out of configtestdata so they can use VFs passed in and not the ones declared in the YAML

**Comment by dkosteck** [comment]
Report of all tests: https://gist.github.com/dkosteck/a6e24b5182aaf7f142839bf3ccc6a657

File: sriov/tests/common/test_utils.py (hunk 1 above)
Comment (jianzzha) [reply sriov/tests/common/test_utils.py:29]: @dkosteck Take a look of how this test was done in PR#145 and let me know what you think

File: sriov/tests/SR_IOV_BondVF_DPDK/test_SR_IOV_BondVF_DPDK.py [hunk 2]
```diff
@@ -56,11 +56,15 @@ def dut_setup(dut, settings, testdata, request) -> Bond:
 
     execute_and_assert(dut, steps, 0, 0.1)
 
-    pci_pf1_vf0 = get_pci_address(dut, pf1 + "v0")
+    pci_pf1_vf0 = get_pci_address(
```
Comment (jianzzha) [inline sriov/tests/SR_IOV_BondVF_DPDK/test_SR_IOV_BondVF_DPDK.py:59]: Liked this change

**Comment by jianzzha** [comment]
@dkosteck This PR is trying to address the same issue as PR#145. So yeah we should merge them. The PR#45 change set is smaller.  I will ping you from PR#45 on the key change there, and we can make a decision which one to go

File: sriov/tests/SR_IOV_BondVF_DPDK/test_SR_IOV_BondVF_DPDK.py (hunk 2 above)
Comment (jianzzha) [reply sriov/tests/SR_IOV_BondVF_DPDK/test_SR_IOV_BondVF_DPDK.py:59]: for history reason, we don't have an interface name for vf1 in the testbed.yaml, for handling consistency, maybe we leave it the way it was and clean this up later in a separate PR where we add the interface name for vf1

**Comment by dkosteck** [comment]
> @dkosteck This PR is trying to address the same issue as PR#145. So yeah we should merge them. The PR#45 change set is smaller. I will ping you from PR#45 on the key change there, and we can make a decision which one to go

@jianzzha Got it, we can sync on next week and I'll create a unified PR

**Comment by dkosteck** [comment]
https://github.com/redhat-partner-solutions/rhel-sriov-test/pull/145 accomplishes this in a simple way, closing to give way to that

**Pull request closed without merging**